src/uu/free/src/free.rs

```diff
@@ -43,10 +48,66 @@ struct MemInfo {
     swap_free: u64,
     swap_used: u64,
     reclaimable: u64,
+    low_total: u64,
+    low_used: u64,
+    low_free: u64,
+    high_total: u64,
+    high_used: u64,
+    high_free: u64,
+    commit_limit: u64,
+    committed: u64,
+}
+
+impl MemInfo {
+    fn make_new(&self, o: &MemInfo, min: bool) -> MemInfo {
+        // if min is true return the smaller value, else return the higher
+        fn compare(a: u64, b: u64, min: bool) -> u64 {
+            if min {
+                a.min(b)
+            } else {
+                a.max(b)
+            }
+        }
+
+        // invert the behaviour for free and avail. memory
+        let clos = |a: u64, b: u64| compare(a, b, min);
```

Comment:
please rename clos and anticlos

like
chooseMinMax
and
reverse_chooseMinMax